README.md

```diff
@@ -120,6 +120,8 @@ Popular block explorers provide a list of active validators. The easiest entry p
 * [List on BigDipper](https://cosmos.bigdipper.live/validators)
 
 DYOR when choosing a validator. Consider delegating your tokens to validators outside of the top 20 to increase the decentralization of the network.
+Check validator voting power on [Kujira POD](https://pod.kujira.app/cosmoshub-4).
```

Comment:
We shouldn't promote a specific service. The services mentioned above can do the same thing. I think it's best to just add it to the list.